### PR TITLE
Based on #93 , bring back rbenv plungins support.

### PR DESCRIPTION
**Pull request by salimane**

### CODE REVIEW

**Comment by seanknox** [comment]
@salimane Like the concept and overall looks good to me. A lot of (seemingly-unrelated) formatting changes to the README however—are those necessary?

**Comment by salimane** [comment]
@seanknox, well, they are not necessary per se, but a nice markdown syntax fix never hurts :)

File: README.md [hunk 1]
```diff
@@ -105,11 +118,6 @@ The following variables may be automatically overridden with Hiera:
   "2.0.0-github": "2.0.0-github6"
 ```
 
-It is **required** that you include
-[ripienaar/puppet-module-data](https://github.com/ripienaar/puppet-module-data)
-in your boxen project, as this module now ships with many pre-defined versions
-and aliases in the `data/` directory. With this module included, those
-definitions will be automatically loaded, but can be overridden easily in your
-own hierarchy.
+It is **required** that you include[ripienaar/puppet-module-data](https://github.com/ripienaar/puppet-module-data) in your boxen project, as this module now ships with many pre-defined versions and aliases in the `data/` directory. With this module included, those definitions will be automatically loaded, but can be overridden easily in your own hierarchy.
```
Comment (seanknox) [inline README.md]: Typo—needs a space before the puppet-module-data link.

**Comment by seanknox** [comment]
@salimane Sure, but a commit should be specific in its scope as well as [document non-obvious, additional changes](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html). Always be [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)'ing. Helps catch typos, like this one: https://github.com/boxen/puppet-ruby/pull/125/files#r22442808

Be sure to rebase too.

**Comment by dgoodlad** [comment]
:+1: for the comments about removing the markdown changes; they're unrelated and make this PR more difficult to parse

> Be sure to rebase too

I don't think that's necessary; it's not the policy of the Boxen project to require a PR to look like a fast-forward. I'd rather not introduce such a rule now.

**Comment by seanknox** [comment]
@dgoodlad Will point out fixing merge conflicts only as opposed to specifically rebasing. Thanks!

**Comment by salimane** [comment]
@seanknox @dgoodlad the markdown changes have been reverted :)

**Comment by salimane** [comment]
any update on this ?
Thanks

**Comment by salimane** [comment]
@dgoodlad can this be merged ?

**Comment by seanknox** [comment]
@salimane this looks good. minor nit: can you update the Ruby version examples to be 2.x instead of 1.9.3? Trying to encourage people to the newer Ruby versions, if only subliminally. :) I'll merge this in at that point.

**Comment by salimane** [comment]
@seanknox done :)

**Comment by seanknox** [comment]
:+1: thanks @salimane!

**Comment by salimane** [comment]
@seanknox can we get a new release tag ?
Thanks

**Comment by seanknox** [comment]
Done: https://github.com/boxen/puppet-ruby/releases/tag/8.5.2

**Comment by dlackty** [comment]
@salimane When I tried this to install rbenv plugins, I keep getting the following error message:

```Error: Could not find dependency File[/opt/boxen/rbenv/plugins] for Repository[/opt/boxen/rbenv/plugins/rbenv-gem-rehash] at /opt/boxen/repo/shared/ruby/manifests/rbenv/plugin.pp:20```

and this is how my site.pp looks like:

```puppet
  # default ruby versions
  ruby::version { '2.2.2': }

  # rbenv plugins
  ruby::rbenv::plugin { 'rbenv-gem-rehash':
    ensure => 'v1.0.0',
    source  => 'sstephenson/rbenv-gem-rehash'
  }
```

Is there anything wrong?

**Comment by salimane** [comment]
@dlackty #141  should fix your problem.
Thanks

**Comment by dlackty** [comment]
@salimane Awesome. Thank you! :+1:

**Comment by blackjid** [comment]
thanks @salimane I added this feature in a https://github.com/boxen/puppet-nodejs/pull/48 rewrite...

**Comment by salimane** [comment]
@blackjid :+1:

**Pull request merged**